docs/source/user-guide/tasks/manage-environments.rst

```diff
@@ -123,6 +123,46 @@ Use the terminal for the following steps:
 
    You can also use ``conda info --envs``.
 
+.. _specifying-environment-platform:
+
+Specifying a different target platform for an environment
+=========================================================
+
+By default, ``conda`` will create environments targeting the platform
+currently running on. You can check which one is yours by running
+``conda info`` and checking the ``platform`` entry.
+
+However, in some cases you might want to create an environment for a
+different target platform or architecture. To do so, use the
+``--platform`` flag available in the ``conda create`` and
+``conda env create`` commands. The value must be a valid platform.
```

Comment:
Do we have a list of valid platforms in the documentation? Would be a nice thing to link to.

Reply:
Looks like our CLI help has them listed. https://docs.conda.io/projects/conda/en/latest/commands/env/create.html#named-arguments

Reply:
Link added.